src/index.html

```diff
@@ -15,13 +15,438 @@
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="rating__review">Reviews: 5</div>
+        </div>
+
+        <div class="cash">
+          <div class="cash__name cash__name--price">Price:</div>
+          <div class="cash__sum cash__sum--value-of-price">$2,199</div>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+        >
+          BUY
+        </a>
+      </div>
+      <div class="card card--7--isActive">
+        <div class="image">
+          <img
+            src="/src/images/imac.jpeg"
+            alt="apple_image"
+            class="image__display image__display--settings"
+          />
+        </div>
+
+        <a
+          href="#"
+          class="card__product"
+        >
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </a>
+
+        <section class="card__code">Product code: 195434</section>
+
+        <div class="rating">
+          <div class="stars">
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="rating__review">Reviews: 5</div>
+        </div>
+
+        <div class="cash">
+          <div class="cash__name cash__name--price">Price:</div>
+          <div class="cash__sum cash__sum--value-of-price">$2,199</div>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+        >
+          BUY
+        </a>
+      </div>
+      <div class="card card--8--isActive">
+        <div class="image">
+          <img
+            src="/src/images/imac.jpeg"
+            alt="apple_image"
+            class="image__display image__display--settings"
+          />
+        </div>
+
+        <a
+          href="#"
+          class="card__product"
+        >
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </a>
+
+        <section class="card__code">Product code: 195434</section>
+
+        <div class="rating">
+          <div class="stars">
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star stars__star--active"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="rating__review">Reviews: 5</div>
+        </div>
+
+        <div class="cash">
+          <div class="cash__name cash__name--price">Price:</div>
+          <div class="cash__sum cash__sum--value-of-price">$2,199</div>
+        </div>
+
+        <a
+          href="#"
+          class="card__button"
+        >
+          BUY
+        </a>
+      </div>
```

Comment:
All product cards use the same image and product details. If this is unintentional, ensure each card has unique content to accurately reflect different products.